scene/animation/animation_mixer.cpp

```diff
@@ -1634,14 +1634,11 @@ void AnimationMixer::_blend_process(double p_delta, bool p_update_only) {
 							default:
 								break;
 						}
-						if (player2->is_playing() || seeked) {
+						if (player2->is_playing()) {
```

Comment:
```suggestion
```
If "seeked" were always true, there would be no need for an if block in the first place.

Reply:
I see, thanks!